python-packages/order_utils/.discharge.json

```diff
@@ -0,0 +1,13 @@
+{
+    "domain": "0x-order-utils-py",
+    "build_command": "/home/gene/.pythonz/bin/python setup.py build_sphinx",
```

Comment:
Why is your home dir in this? Will it work for others on the team?

Reply:
Hah, sorry, last minute hacking let this creep in.  If you're curious, it's because my bleeding-edge Python build is installed for my user only, not system-wide, and `discharge` uses the system one, so I had to hack this in.  I'll put in a non-qualified invocation.